### DISCUSSION TITLE
When was the first M&M's produced?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#832] [Galaxy Trivia] [answered]
**Bonus:** What is the name of the oldest chocolate bars produced by the manufacturer?

**Comment by peterbaccaro** [comment, marked as answer]
The first M&M's were produced in **1941**. They were introduced by Mars, Incorporated, a company founded by Forrest Mars Sr., who developed the candy-coated chocolates in collaboration with Bruce Murrie. The candy got its name from the initials of its inventors, "Mars" and "Murrie". M&M's were initially created as a convenient chocolate treat for soldiers during World War II due to their durable candy coating, which prevented the chocolates from melting easily.

The oldest chocolate bar produced by Mars, Inc. is called "**Mars Bar**". It was first created by the company's founder, Franklin Clarence Mars, in 1932. The Mars Bar consists of nougat, caramel, and roasted almonds, all covered in milk chocolate. It has been a popular chocolate bar for many decades and is known for its rich and indulgent flavour. It should be noted that the Mars Bar has undergone some changes and variations in different regions over time. In the United States, the Mars Bar was discontinued in 2002 and was later reintroduced as the "Snickers Almond Bar", which is a variant of the original Mars Bar. However, in other countries, such as the United Kingdom and Australia, the Mars Bar continues to be sold under its original name.

**Reply by ParanoidUser** [reply]
That's funny, I didn't know the original Mars had almonds. But now taking a closer look at Nuts (hazelnut) and Sneakers (peanut), it seems obvious. 🤓

**Comment by anjutares** [comment]
The first M&M's were produced in 1941. They were created by Forrest Mars Sr., son of the Mars Company founder, and Bruce Murrie, son of Hershey's president at the time, as a convenient way to provide soldiers with chocolate that wouldn't melt during World War II.

As for the oldest chocolate bars produced by the manufacturer, it depends on which manufacturer you are referring to. If you are referring to Mars, Incorporated, the manufacturer of M&M's, one of their oldest chocolate bars is the "Mars" bar, which was first introduced in 1932. It consists of nougat and caramel covered in milk chocolate.

However, if you are referring to a different manufacturer, please specify so that I can provide you with the accurate information.
